lib/bundler/cli/change.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module Bundler
+  class CLI::Change
+    def initialize(options, gem_name)
+      @gem_name = gem_name
+      @options = options
+    end
+
+    def run
+      raise InvalidOption, "Please supply at least one option to change." unless @options[:group] || @options[:version] || @options[:source]
+
+      definition = Bundler.definition
+
+      @dep = definition.dependencies.find {|d| d.name == @gem_name }
+
+      raise InvalidOption, "`#{@gem_name}` could not be found in the Gemfile." unless @dep
+
+      @pass_options = {}
```

Comment:
maybe call this `add_options`, and instead of making it an ivar, pass it into `set_group_options` / `set_version_options`